controllers/managedocs_controller.go

```diff
@@ -1507,21 +1507,26 @@ func (r *ManagedOCSReconciler) reconcileEgressNetworkPolicy() error {
 				return fmt.Errorf("Unable to determine machine CIDR from AWS ConfigMap")
 			}
 			// Allow egress traffic to that cidr range
-			vpcEgressRule := openshiftv1.EgressNetworkPolicyRule{
-				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
-				To: openshiftv1.EgressNetworkPolicyPeer{
-					CIDRSelector: vpcCIDR,
-				},
+			var vpcEgressRules []openshiftv1.EgressNetworkPolicyRule
+			cidrList := strings.Split(vpcCIDR, ";")
+			for _, cidr := range cidrList {
+				if cidr == "" {
+					continue
+				}
+				vpcEgressRules = append(vpcEgressRules, openshiftv1.EgressNetworkPolicyRule{
+					Type: openshiftv1.EgressNetworkPolicyRuleAllow,
+					To: openshiftv1.EgressNetworkPolicyPeer{
+						CIDRSelector: cidr,
+					},
+				})
```

Comment:
No need to use continue (a execution break) for such a simple case
```suggestion
				if cidr != "" {
					vpcEgressRules = append(vpcEgressRules, openshiftv1.EgressNetworkPolicyRule{
						Type: openshiftv1.EgressNetworkPolicyRuleAllow,
						To: openshiftv1.EgressNetworkPolicyPeer{
							CIDRSelector: cidr,
						},
					})
				}
```